createdisk-library.sh

```diff
@@ -209,12 +209,22 @@ EOF
 function prepare_qemu_guest_agent() {
     local vm_ip=$1
 
-    # f36 default selinux policy blocks usage of qemu-guest-agent over vsock
-    ${SCP} qemuga-vsock.te core@${vm_ip}:
-    ${SSH} core@${vm_ip} '/usr/bin/checkmodule -M -m -o qemuga-vsock.mod qemuga-vsock.te'
-    ${SSH} core@${vm_ip} '/usr/bin/semodule_package -o qemuga-vsock.pp -m qemuga-vsock.mod'
+    # f36+ default selinux policy blocks usage of qemu-guest-agent over vsock, we have to install
+    # our own selinux rules to allow this.
+    #
+    # we need to disable pipefail for the `checkmodule | grep check` as we expect `checkmodule`
+    # to fail on rhel8.
+    set +o pipefail
+    if ! checkmodule -c 19 2>&1 |grep 'invalid option' >/dev/null; then
+	    # RHEL8 checkmodule does not have this arg
+	    MOD_VERSION_ARG="-c 19"
+    fi
+    set -o pipefail
+    /usr/bin/checkmodule ${MOD_VERSION_ARG} -M -m -o qemuga-vsock.mod qemuga-vsock.te
```

Comment:
@cfergeau It is now fail with 
```
createdisk-library.sh: line 223: MOD_VERSION_ARG: unbound variable
Build step 'Execute shell' marked build as failure
Finished: FAILURE
```
So we do need a default value for this variable, like empty string.